xcixor/iReporter
# Chore refactor code 162366580

## What is the Purpose of this PR
The purpose of this PR is to refactor code
## Why is it important
To enhance the app's maintainability
## How can it be tested 
switch the branch on [@here](https://github.com/xcixor/iReporter) to chore-refactor-code-162366580 and checkout the readme the codeclimate badge should have improved
## Relevant PT story
[#162366580](https://www.pivotaltracker.com/story/show/162366580)
